GA-worksheet.md

```diff
@@ -6,18 +6,13 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag?Upper Part or a theme or like a title
+2. What is a Navigation Bar?It's like a link to show something imporatant or important part for different parts
+3. What tag do we use to make a navigation bar? <      ><ul> <ol> <li> <ul> Stands for unorderlist <li> Stands for list item
 
-2. What is a Navigation Bar?
+5. What does the <a> tag do?It's like a placeholder or a hyperlink
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
-
-5. What does the <a> tag do?
-
-6. What would the line of HTML below do?
+6. What would the line of HTML below do? Give us the link for somewhere
```

Comment:
Yes! The link below takes us to http://www.google.com.

The `<a>` tag sets up the link and the `href' attribute gives us the destination of the link. In between the `<a>` tags is the text that will serve as the hyperlink.